nautobot/dcim/templates/dcim/device.html

```diff
@@ -66,6 +66,14 @@
                                             <span><a href="{% url 'dcim:devicetype' pk=object.device_type.pk %}">{{ object.device_type.display }}</a> ({{ object.device_type.u_height }}U)</span>
                                         </td>
                                     </tr>
+                                    {% if object.device_type.hardware_family %}
+                                        <tr>
+                                            <td>Hardware Family</td>
+                                            <td>
+                                                <a href="{% url 'dcim:hardwarefamily' pk=object.device_type.hardware_family.pk %}">{{ object.device_type.hardware_family }}</a>
+                                            </td>
+                                        </tr>
+                                    {% endif %}
```

Comment:
```suggestion
                                    <tr>
                                        <td>Hardware Family</td>
                                        <td>{{ object.device_type.hardware_family|hyperlinked_object }}</td>
                                        </td>
                                    </tr>
```

Also, I think it would flow better logically if `Hardware Family` was above `Device Type` rather than below it.